packages/frontend/src/plugins/authHelpers.js

```diff
@@ -103,13 +102,13 @@ export async function signOut(mixpanelInstance) {
   window.location = '/'
 
   if (mixpanelInstance) {
-    mixpanelInstance.track('Log Out', { type: 'action'  })
+    mixpanelInstance.track('Log Out', { type: 'action' })
```

Comment:
we should start a mixpanel constants like you did for the local storage. I like that.

Reply:
yea, good idea